.github/workflows/workflow.yml

```diff
@@ -18,9 +13,6 @@ jobs:
     - name: Checkout master
       uses: actions/checkout@main
 
-    - name: Update SHA
```

Comment:
This is failing as it the directory does not exist. It is part of the static site example and not needed for the python app.

https://github.com/sharmita3/sample-python/runs/1481140544?check_suite_focus=true#step:3:4